contracts/src/spaces/facets/owner/SpaceOwner.sol

```diff
@@ -84,6 +84,10 @@ contract SpaceOwner is
   ) external {
     _onlySpaceOwner(space);
     _updateSpace(space, name, uri, shortDescription, longDescription);
+
+    space.call(abi.encodeCall(MembershipMetadata.refreshMetadata, ()));
```

Comment:
`IMembershipMetadata(space).refreshMetadata()`

Reply:
It will break unit tests because a random space address is used in `SpaceOwnerTest`.

Reply:
Update to use a mock space